features/scaled-nyxus/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim
+
+COPY VERSION /
+		
+ARG EXEC_DIR="/opt/executables"
+ARG DATA_DIR="/data"
+
+RUN mkdir -p ${EXEC_DIR} \
+    && mkdir -p ${DATA_DIR}/inputs \
+    && mkdir ${DATA_DIR}/outputs
+
+COPY src ${EXEC_DIR}/
+WORKDIR ${EXEC_DIR}
+
+RUN pip3 install -r ${EXEC_DIR}/requirements.txt --no-cache-dir --force
+
+ENTRYPOINT ["python3", "main.py"]
```

Comment:
Could we use `CMD` instead of `ENTRYPOINT`? In the same way it is used [here](https://github.com/kannon92/polus-plugins/blob/compute/formats/polus-ome-zarr-converter-plugin/Dockerfile)